applications/lanars/lanars.md

```diff
@@ -0,0 +1,29 @@
+# Lanars
+
+https://lanars.com
+
+`23` [**Березовський** Олександр](/players/lanars/berezovskyi.oleksandr.19870503.jpg) `/1987-05-03/`
```

Comment:
Wrong path
Use `/players/berezovskyi.oleksandr.19870503.jpg` instead of `/players/lanars/berezovskyi.oleksandr.19870503.jpg`
General note: do not use %company-name in the path

Reply:
@ws45